README.md

```diff
@@ -3,5 +3,7 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/4b0a3366dcff40f7be1ad3bf0fe24d30)](https://app.codacy.com/gh/blakes24/dive-atlas?utm_source=github.com&utm_medium=referral&utm_content=blakes24/dive-atlas&utm_campaign=Badge_Grade_Settings)
 [![Coverage Status](https://coveralls.io/repos/github/blakes24/dive-atlas/badge.svg?branch=main)](https://coveralls.io/github/blakes24/dive-atlas?branch=main)
 
+Branch specific badge: [![Coverage Status](https://coveralls.io/repos/github/blakes24/dive-atlas/badge.svg?branch=fix-badge)](https://coveralls.io/github/blakes24/dive-atlas?branch=fix-badge)
+
```

Comment:
What do you mean by "Branch specific badge" ?

Reply:
The badges are associated with a specific branch. This one is for the fix-badge branch so I could see if it was displaying the percentage. The top one is for the main branch but it won't update until travis does a build of the main branch.